cmd/zc_traverser_blob.go

```diff
@@ -130,13 +131,19 @@ func (t *blobTraverser) traverse(processor objectProcessor, filters []objectFilt
 				continue
 			}
 
+			encoding := ""
```

Comment:
Note for sometime after the s2s refactor gets merged in: use common.IffStringNotNil here instead.